dnnsoftware/Dnn.Platform
# Add validation method with IsNullOrWhiteSpace check.

**Test Steps**

1. Login with superuser
2. Persona Bar --> Roles --> Create New role
3. Enter a Role name
4. From the Role Group dropdown --> Select New group
5. Enter Blank name (Enter Space)
6. Input a blank description (Space(s))
7. Click Save
**Expected Result:** Error message should be displayed --> The argument 'Name' cannot be null or empty. Parameter name: Name

Fixes #2927 

## Summary
Added validation method to determine whether the argValue is null, empty, or has whitespace only. Also updated the RoleGroup Name validation to reference the new validation method.
